src/main/java/stsjorbsmod/powers/SacrificePower.java

```diff
@@ -42,4 +44,22 @@ public void updateDescription() {
     public AbstractPower makeCopy() {
         return new SacrificePower((AbstractPlayer) owner, this.amount);
     }
+
+    @Override
+    public int getAlternateResource(AbstractCard c) {
+        if (c.cost == -1)
+            return EnergyPanel.totalCount;
+        return owner.maxHealth;
+    }
+
+    @Override
+    public int spendAlternateCost(AbstractCard c, int i) {
+        addToBot(new DecreaseMaxHpAction(owner, owner, i, AbstractGameAction.AttackEffect.POISON));
```

Comment:
I'm not sure how much I like the losing Max Hp as a part of an action. Does the player lose Max HP if the combat ends?

Reply:
I think this is how we had it before as well as the other cards that affect Max HP, see https://github.com/dbjorge/jorbs-spire-mod/blob/master/src/main/java/stsjorbsmod/patches/SacrificeEnergyPatch.java

Play does lose the Max HP if their attack deals a killing blow, which I think is the intended effect. The player would still lose energy if that attack dealt a killing blow.

Obviously open to changes, but I was just doing a direct port over using the BaseMod features.

Reply:
As long as the player loses Max HP from the card play, then I'm good.